src/cablab/providers/air_temperature.py

```diff
@@ -29,7 +27,7 @@ def get_variable_descriptors(self):
         return {
             VAR_NAME: {
                 'data_type': numpy.float32,
-                'fill_value': -32767,
+                'fill_value': FILL_VALUE,
                 'units': 'K',
                 'long_name': '2 metre temperature',
                 'scale_factor': 0.0019718202938428923,
```

Comment:
Once you are at it, could you try to remove the scale_factor and offset so that we have air temperature in Celsius and don't have to do any math when reading the data into the cube? Not all NetCDF packages in all languages automatically apply the offset and scale_factor when reading in data.